float64[] setpoint
uint8 type
float64 time
uint8 TYPE_JOINT_POSITION=1
uint8 TYPE_JOINT_VELOCITY=2
uint8 TYPE_CARTESIAN_POSITION=3
uint8 TYPE_CARTESIAN_VELOCITY=4
uint8 TYPE_JOINT_TORQUE=5
uint8 TYPE_TOOL_FORCE=6
uint8 TYPE_CARTESIAN_POSITION_IMPEDANCE=7
